src/Libraries/DSOfficeUtilities/OpenXmlHelper.cs

```diff
@@ -0,0 +1,656 @@
+        {
+            if (writeAsString)
+            {
+                value = Convert.ToString(value, CultureInfo.InvariantCulture);
+            }
+
+            if (value is string)
+            {
+                // For strings, we put them in the shared string table and reference by index, just like Office would
+                var tuple = sharedStringTable.Select((v, i) => System.Tuple.Create(v, i)).FirstOrDefault(t => t.Item1.InnerText.Equals(value));
+                int index;
+                if (tuple == null)
+                {
+                    // The string is not in the table, so we add it
+                    sharedStringTable.AppendChild(new SharedStringItem(new Text((string)value)));
+                    index = (int)sharedStringTable.Count.Value;
+                    // Yes, you need to update these manually
+                    sharedStringTable.Count++;
+                    sharedStringTable.UniqueCount++;
+                }
+                else
+                {
+                    index = tuple.Item2;
+                }
+
+                cell.CellValue = new CellValue(index.ToString(CultureInfo.InvariantCulture));
+                cell.DataType = CellValues.SharedString;
+            }
+            else if (value is bool)
+            {
+                cell.CellValue = new CellValue(Convert.ToInt32((bool)value).ToString(CultureInfo.InvariantCulture));
+                cell.DataType = CellValues.Boolean;
+            }
+            else if (value is DateTime)
+            {
+                const int DateTimeFormatId = 22;
+                // Search for a style using the DateTime format id
+                var tuple = stylesheet.CellFormats.Select((v, i) => System.Tuple.Create((CellFormat)v, i)).FirstOrDefault(p => p.Item1.NumberFormatId != null && p.Item1.NumberFormatId.Value == DateTimeFormatId);
+                uint index;
+                if (tuple == null)
+                {
+                    // No such style exists so we create one
+                    stylesheet.CellFormats.AppendChild(new CellFormat() { NumberFormatId = DateTimeFormatId, FormatId = 0 });
+                    index = stylesheet.CellFormats.Count.Value;
+                    // We update this for the sake of consistency
+                    stylesheet.CellFormats.Count++;
+                }
+                else
+                {
+                    index = (uint)tuple.Item2;
+                }
+
+                cell.StyleIndex = index;
+                cell.CellValue = new CellValue(((DateTime)value).ToOADate());
+                cell.DataType = null;
+            }
+            else
+            {
+                // This is for long and double. Also acts as the default behavior for things we do not specially handle.
+                cell.CellValue = new CellValue(Convert.ToString(value, CultureInfo.InvariantCulture));
+                cell.DataType = null;
+            }
+        }
+    }
+}
```

Comment:
So all of this code doesn't duplicate what's already there in `DSOffice` as `DSOffice `isn't included the .net6 sln and the `DSOfficeUtilities` is not included in the net48 sln correct?